05-descriptive-analysis.Rmd

```diff
@@ -249,17 +251,17 @@ recs_des %>% #  Specify the sample design
   survey_count() # Run `survey_count()`
 ```
 
-There are 118,208,250 observations in the survey. <!--TODO: Calculate this-->
+There are 118,208,250 observations in the survey. <!--TODO: Calculate this-->    
 
-`srvyr::count()` can take one or many variables. To calculate the number of observations for a combination of variables, such as Region and Division, we run the below:
+`srvyr::count()` can take one or many variables. To calculate the number of observations for a combination of variables, such as Region and Division, we run the below:    
 
 ```{r}
 recs_des %>%
   # Specify the required arguments within the function
   survey_count(Region, Division, name = "N") 
 ```
 
-When we run the crosstab, we see 5,638,844 observations from the Northeast's New England division, 15,377,694 from the Northeast's Middle Atlantic division, and so on. <!--TODO: Calculate this-->
+When we run the crosstab, we see 5,638,844 observations from the Northeast's New England division, 15,377,694 from the Northeast's Middle Atlantic division, and so on. <!--TODO: Calculate this-->   
```

Comment:
```suggestion
When we run the crosstab, we see 5,638,844 weighted observations from the Northeast's New England division, 15,377,694 from the Northeast's Middle Atlantic division, and so on. <!--TODO: Calculate this-->   
```

Reply:
I would call this the estimated number of households from the Northeast's New....